app/components/sections/price-tiers/price-tier-editor/component.js

```diff
@@ -2,13 +2,13 @@ import Ember from 'ember';
 import computed from 'ember-computed-decorators';
 
 export default Ember.Component.extend({
-  @computed('model.itemPrices.@each.{isNew}')
+  @computed('model.itemPrices.@each.{isNew,item}')
   openItemPrices(itemPrices = []) {
-    return itemPrices.filter(ip => ip.get('isNew'));
+    return itemPrices.filter(ip => ip.get('isNew') && ip.get("item.active"));
```

Comment:
Since active is not a key above this won't update if we ever have a situation where the active prop changes.

Instead of this we should move the `isNew` and `item.active` into a new computed prop on the `itemPrice` model directly. Something like, `itemPrice.isPending` for the `isNew` and `item.active`

and `itemPrice.isActive` for `!isNew` and `item.active`

Let me know what you think